includes/external-connection-cpt.php

```diff
@@ -516,6 +526,34 @@ function meta_box_external_connection_details( $post ) {
 		<ul class="endpoint-errors"></ul>
 	</div>
 
+	<?php if ( ! empty( $external_connection_status ) ) : ?>
+	<div class="post-types-permissions hide-until-authed">
```

Comment:
Minor thing but this block should probably be indented within this `if` statement